content/zh/docs/contribute/localization_zh.md

```diff
@@ -0,0 +1,463 @@
+```
+
+#### Frontmatter 的处理
+
+页面中的 Frontmatter 指的是文件头的两个 `---` 中间的部分。
+对这一部分，解析器有特殊处理，因此不能将英文部分放在前面，中文跟在后面。
+需要将二者顺序颠倒。如下所示：
+
+```
+---
+title: 译文标题
+type: concept
+weight: 30
+---
+
+<!--
+title: English title
+type: concept
+reviewers:
+  - john
+  - doe
+weight: 30
+-->
+```
+
+这里要注意的是：
+
+- `title`、`description` 的内容要翻译，其他字段一般不必（甚至不可）翻译。
+- `reviewers` 部分要删除，不然中文译文会转给英文作者来审阅。
+
+
+#### 短代码（shortcode）处理
+
+通过 HTML 注释的短代码仍会被运行，因此需要额外小心。建议处理方式：
+
+```
+<!--
+English text
+-->
+{{</* note */>}}
+中文译文
+{{</* /note */>}}
+```
+
+评阅人应该不难理解中英文段落的对应关系。但是如果采用下面的方式，则会出现
+两个 `note`，因此需要避免。这是因为被注释起来的短代码仍会起作用！
+
+```
+<!--
+{{</* note */>}}
+English text
+{{</* /note */>}}
+-->
+{{</* note */>}}
+中文译文
+{{</* /note */>}}
+```
+
+### 译与不译
+
+#### 资源名称或字段不译
+
+根据英文原文写作风格约定【也在持续修订改进】，对 Kubernetes 中的 API
+资源均按其规范中所给的大小写形式书写，例如：英文中会使用 Deployment 而不是
+deployment 来表示名为 "Deployment" 的 API 资源类型和对象实例。
+
+对这类词语，一般不应翻译。
+
+{{< note >}}
+英文原文在这方面并不严谨，译者或中文译文的评阅者要非常留心。
+比如 Secret 资源，很多时候被误写为 secret。这时在本地化版本中一定
+不能译为“秘密”，以免与原文的语义不符。
+{{< /note >}}
+
+#### 代码中的注释
+
+一般而言，代码中的注释需要翻译，包括存放在 `content/zh/examples/` 目录下
+的清单文件中的注释。
+
+
+#### 出站链接
+
+如果超级链接的目标是 Kubernetes 网站之外的纯英文网页，链接中的内容*可以*不翻译。
+例如：
+
+```
+<!--
+Please check [installation caveats](https://acme.com/docs/v1/caveats) ...
+-->
+请参阅 [installation caveats](https://acme.com/docs/v1/caveats) ...
+```
+
+注意，这里的 `installation` 与 `参阅` 之间留白，因为解析后属于中英文混排的情况。
+
+### 标点符号
+
+译文中标点符号要使用全角字符，除非以下两种情况：
+
+- 标点符号是英文命令的一部分；
+- 标点符号是 Markdown 语法的一部分。
+
+英文排比句式中采用的逗号，在译文中要使用顿号代替，复合中文书写习惯。
+
+## 关于链接
+
+### 链接锚点
+
+英文 Markdown 中的各级标题会自动生成锚点，以便从其他页面中链接。
+在译为中文后，相应的链接必然会失效。为防止这类问题，建议在翻译
+各级标题时，使用英文方式显式给出链接锚点。例如：
+
+```
+<!--
+### Create a Pod
+-->
+### 创建 Pod   {#create-a-pod}
+```
+
+此类问题对于概念部分的页面最为突出，需要格外注意。
+
+
+### 中文链接目标
+
+由于大部分页面已经完成中文本地化，这意味着很多链接可以使用中文版本作为目标。
+例如：
+
+```
+<!--
+For more information, please check [volumes](/docs/concepts/storage/)
+...
+-->
+更多的信息可参考[卷](/zh/docs/concepts/storage/)页面。
+```
+
+如果对应目标页面尚未本地化，建议登记一个 Issue。
+
+{{< note >}}
+Website 的仓库中 `scripts/linkchecker.py` 是一个工具，可用来检查页面中的链接。
+例如，下面的命令检查中文本地化目录 `/content/zh/docs/concepts/containers/`
+中所有 Markdown 文件中的链接合法性：
+
+```
+./scripts/linkchecker.py -l zh -f /docs/concepts/containers/**/*.md
+```
+{{< /note >}}
+
+## 排版格式
+
+以下为译文 Markdown 排版格式要求：
+
+- 中英文之间留一个空格
+  * 这里的“英文”包括以英文呈现的超级链接
+  * 这里的中文、英文都不包括标点符号
+- 译文 Markdown 中不要使用长行，应适当断行。
+  * 可根据需要在 80-120 列断行
+  * 最好结合句子的边界断行，即一句话在一行，不必留几个字转到下一行
+  * 超级链接文字一般较长，可独立成行
+
+{{< warning >}}
+我们注意到有些贡献者可能使用了某种自动化工具，在 Markdown 英文原文中自动
+添加空格。虽然这些工具可一定程度提高效率，仍然需要提请作者注意，某些工具
+所作的转换可能是不对的，例如将 `foo=bar` 转换为 `foo = bar`、将
+`），另一些文字` 转换为 `） ，另一些文字` 等等，甚至将超级链接中的半角
+井号（`#`）转换为全角，导致链接失效。
+{{< /warning >}}
+
+英文中 "you" 翻译成 "你" 不必是 “您"
+文章内的链接用英文例如 (#deploying )，在对应的标题上后面加上 {#deploying}
+
+
+## 术语
+
+### 术语拼写
+
+按中文译文习惯，尽量不要在中文译文中使用首字母小写的拼写。例如：
+
+```
+列举所有 pods，查看其创建时间 ...       [No]
+列举所有 Pod，查看其创建时间 ...        [Yes]
+```
+
+*第一次*使用首字母缩写时，应标注其全称和中文译文。例如：
+
+```
+你可以创建一个 Pod 干扰预算（Pod Disruption Budge，PDB）来解决这一问题。
```

Comment:
```suggestion
你可以创建一个 Pod 干扰预算（Pod Disruption Budget，PDB）来解决这一问题。
```
?
Although readers may also want to know the name as seen in the API, eg PodDisruptionBudget.